LICENSE.txt

```diff
@@ -1,13 +1,27 @@
-BSD 3-clause license
-Copyright (c) 2015-2020, conda-forge contributors
+BSD-3-Clause license
+Copyright (c) 2015-2022, conda-forge contributors
```

Comment:
SHould this be up to 2024?

Reply:
This is a copy of the license for conda-forge itself, and interestingly enough the latest public conda-forge license says "2015-2022". Here is the header on the copy that appears in the numpy feedstock, for example:

> BSD-3-Clause license
> Copyright (c) 2015-2022, conda-forge contributors
> All rights reserved.

https://github.com/conda-forge/numpy-feedstock/blob/main/LICENSE.txt